tests/Query/WildcardTest.php

```diff
@@ -106,4 +106,31 @@ public function testSearchWithAnalyzer(): void
 
         $this->assertEquals(0, $resultSet->count());
     }
+
+    /**
+     * @group functional
+     */
+    public function testCaseInsensitive(): void
+    {
+        // feature doesn't exist on version prior 7.10;
+        $this->_checkVersion('7.10');
+
+        foreach ([true, false] as $expected) {
```

Comment:
Can we use a @dataprovider annotation instead of having a foreach?

Reply:
I've changed it.